odoo/addons/base/models/qweb.py

```diff
@@ -794,15 +609,40 @@ def _is_static_node(self, el, options):
         """ Test whether the given element is purely static, i.e., does not
         require dynamic rendering for its attributes.
         """
-        return not any(att.startswith('t-') for att in el.attrib)
+        return el.tag != 't' and not any(att.startswith('t-') and att not in ['t-tag', 't-content'] for att in el.attrib)
 
     # compile
 
-    def _compile_static_node(self, el, options):
+    def _compile_hook_before_directive(self, el, directive, options, indent):
+        if 'profile' not in options:
+            return []
+
+        path = el is not None and options['root'].getpath(el) or ""
+        loginfo = re.sub(_VARNAME_REGEX, '_', 'loginfo_%s_%s' % (path, directive))
```

Comment:
_VARNAME_REGEX is a regular expression object, so it has method `sub`:
```suggestion
        loginfo = _VARNAME_REGEX.sub('_', 'loginfo_%s_%s' % (path, directive))
```